pkg/index/index.go

```diff
@@ -167,6 +167,17 @@ func (f FloatTermValue) String() string {
 	return strconv.FormatInt(numeric.Float64ToInt64(f.Value), 10)
 }
 
+// IntTermValue represents a int term value.
```

Comment:
It's not a general IntTermValue; it is only used by the timestamp. How about renaming it to "TimeStampValue"?